easybuild/easyconfigs/h/hivtrace/hivtrace-0.6.2-foss-2021a.eb

```diff
@@ -0,0 +1,72 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Iñaki Mtz de Ilarduya Muñoz
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonBundle'
+
+name = 'hivtrace'
+version = '0.6.2'
+
+homepage = 'https://github.com/veg/hivtrace'
+description = """HIV-TRACE is an application that identifies potential transmission clusters within a
+ supplied FASTA file with an option to find potential links against the Los Alamos HIV Sequence Database."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('Biopython', '1.79'),
+    ('tn93', '1.0.7'),
```

Comment:
@i-mtz can you add the easyconfig for this software?

Reply:
@smoors  Should I add it in this PR or as a separate one?

Reply:
@i-mtz you can choose, both is ok, but it's probably easier to just add it here.
if you do it in a separate PR, make sure to mention it at the top of this PR.